### DISCUSSION TITLE
Github Copilot Suggesting Webhook URL

### DISCUSSION THREAD

**Discussion by ywaf** [community/community#15672] [Copilot Conversations]
Hello, Was using GitHub copilot with PyCharm Pro (python 3) when I noticed that copilot was suggesting a webhook url (note the screenshot is NOT my webhook and I havent entered any other webhooks in the project). I believe this to be a security (?) issue. 

Please lmk if this isnt the correct place to post this.

Thanks,
Leho

![pycharm64_vzmH4nvlXZ](https://user-images.githubusercontent.com/52742690/165505975-67d9053c-fe8e-4e62-9a5a-fa24395988bd.png)

![pycharm64_JLvHV8744q](https://user-images.githubusercontent.com/52742690/165506166-76771753-16bb-465a-bca7-e14be7484f17.png)

**Comment by ghost** [comment]
the webhooks are composed and random. if you try connecting to them you wouldn't be able to.

I had a similar thing recently when it started suggesting youtube video url's but all of them (over 30) all were invalid
